testData/solutions/web.json

```diff
@@ -164,5 +164,131 @@
+                                }
+                            }
+                        }
+                    },
+                    "links": {
+                        "transform": {
+                            "type": "fluid.transforms.valueMapper",
+                            "inputPath": "http://registry\\.gpii\\.net/common/linkAdaptation",
+                            "options": {
+                                "emphasizeLinks": {
+                                    "outputValue": true
+                                }
+                            }
+                        }
+                    },
+                    "contrastTheme": {
+                        "transform": {
+                            "type": "fluid.transforms.condition",
+                            "conditionPath": "http://registry\\.gpii\\.net/common/highContrastEnabled",
+                            "true": {
+                                "transform": {
+                                    "type": "fluid.transforms.valueMapper",
+                                    "inputPath": "http://registry\\.gpii\\.net/common/highContrastTheme",
+                                    "defaultInputValue": "default",
+                                    "options": {
+                                        "black-white": {
+                                            "outputValue": "bw"
+                                        },
+                                        "white-black": {
+                                            "outputValue": "wb"
+                                        },
+                                        "black-yellow": {
+                                            "outputValue": "by"
+                                        },
+                                        "yellow-black": {
+                                            "outputValue": "yb"
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    },
+                    "signLanguageEnabled": "http://registry\\.gpii\\.net/common/signLanguageEnabled",
+                    "signLanguage": {
+                        "transform": {
+                            "type": "fluid.transforms.valueMapper",
+                            "inputPath": "http://registry\\.gpii\\.net/common/signLanguage",
+                            "options": {
+                                "ase": {
+                                    "outputValue": "ase"
+                                },
+                                "gsg": {
+                                    "outputValue": "gsg"
+                                },
+                                "ils": {
+                                    "outputValue": "ils"
+                                }
+                            }
+                        }
+                    },
```

Comment:
The OLB only supports three sign languages, but the value range of the common term "signLanguage" is all sign languages in the world, so a prefs set with the common term "signLanguage" can very well have a value that the OLB does not support. I can see two ways of dealing with this:
1. at the level of the transformer, where the valueMapper might use "ils" (International Sign) as a fallback for sign languages that the OLB does not support;
2. at the level of the web app (assuming we change the transformer to "signLanguage": "http://registry\.gpii\.net/common/signLanguage"), which would itself deal with unsupported values. 

But my understanding of the Matchmaking (and transformation) process is that we avoid sending unsupported values to the application if we know in advance that they are not supported. So I'm more in favour of the first approach, but I couldn't figure out how to make the transformer use "ils" as a fallback.